src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -329,6 +329,15 @@
                     ]
                 }
             ]
+        },
+        {
+            "Id": "CNamePrefix",
+            "Name": "CName Prefix",
+            "Description": "If specified, the environment attempts to use this value as the prefix for the CNAME in your Elastic Beanstalk environment URL. If not specified, the CNAME is generated automatically by appending a random alphanumeric string to the environment name.",
+            "Type": "String",
+            "DefaultValue": "",
+            "AdvancedSetting": true,
+            "Updatable": false
```

Comment:
I know we have a separate story for missing validations, but I'd prefer you add any existing validations that may apply. Since you already know the purpose of a `CNamePrefix`, refer to the documentation and add any Regex validator that is applicable.

Reply:
The documentation does not specify any restrictions to the `CNamePrefix`. From my testing , I didn't run into any restrictions so I wouldn't want to add a Regex to restrict the values beyond what is allowed.